note/answers/chapter12_classes.md

```diff
@@ -145,8 +145,8 @@
         parameters = new ArrayList<>();
         if (!check(RIGHT_PAREN)) {
           do {
-            if (parameters.size() >= 8) {
-              error(peek(), "Can't have more than 8 parameters.");
+            if (parameters.size() >= 255) {
```

Comment:
```suggestion
            if (parameters.size() >= 256) {
```

Reply:
[But it is `>=` in the book](https://github.com/munificent/craftinginterpreters/blob/6eebbecf5660e3bc9234645adcdddafc968dfc5c/java/com/craftinginterpreters/lox/Parser.java#L260)

Reply:
Then that also needs to be changed since the message and the check don't agree.

Reply:
The code is a little confusing. Note that the check occurs *before* the next parameter is parsed. So if `parameters.length` is 255, then it will overflow shortly because it is about the parse the next parameter.